docs/rules/no-shadow-native-events.md

```diff
@@ -0,0 +1,51 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-shadow-native-events
+description: disallow `emits` which would shadow native html events
```

Comment:
Can you please run `npm run update` to update the rule docs with the description from the rule source itself?

Reply:
done